resources/language/Portuguese/strings.po

```diff
@@ -212,11 +215,12 @@ msgid "Resolution preference for TV Shows"
 msgstr "Preferência de resolução para Séries"
 
 msgctxt "#30049"
-msgid "Percentage of additional seeders required before choosing next resolution"
-msgstr "Percentagem de seeders adicionais antes de ser escolhida a proxima resolução"
+msgid ""
+"Percentage of additional seeders required before choosing next resolution"
+msgstr ""
+"Percentagem de seeders adicionais antes de ser escolhida a proxima resolução"
```

Comment:
This is valid PO format. The PO editor I used breaks long lines this way.
See end of [gettext manual](https://www.gnu.org/software/gettext/manual/html_node/PO-Files.html) for an example and [Pology manual](http://pology.nedohodnik.net/doc/user/en_US/ch-poformat.html#sec-powrap).